.travis.yml

```diff
@@ -247,7 +247,11 @@ before_install:
 install:
     - if [[ $USE_CI_HELPERS == True ]]; then
         git clone git://github.com/astropy/ci-helpers.git;
-        source ci-helpers/travis/setup_conda.sh;
+        if [[ $TRAVIS_OS_NAME == osx || $TRAVIS_OS_NAME == windows ]]; then
+          source ci-helpers/travis/setup_python.sh;
```

Comment:
This won't actually install PIP_DEPENDENCIES so might need to add a ``pip install $PIP_DEPENDENCIES`` below?

Reply:
Done. Thanks!